content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1063,6 +1123,13 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+{{< note >}}
+Since v1.31, when you enable the `JobPodReplacementPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+(enabled by default), the Job controller awaits for termination of all pods
+before marking a Job as terminal (see [Termination of Job Pods](#termination-of-job-pods)).
+{{< /note >}}
```

Comment:
Not sure how this relates to the replacement of failed Pods. I would remove this, since it's a bit confusing here

Reply:
I agree, it can be confusing. The only purpose of this was to reference that the logic change is controlled by the feature gate. Now I propose to mention this as a note in one place only.